src/System/NowPlaying.m

```diff
@@ -155,12 +170,20 @@ - (void)updateInfo
             NSString *album = [info objectForKey:kMRMediaRemoteNowPlayingInfoAlbum];
             NSString *artist = [info objectForKey:kMRMediaRemoteNowPlayingInfoArtist];
             NSString *title = [info objectForKey:kMRMediaRemoteNowPlayingInfoTitle];
-
-            if (self.album != album || self.artist != artist || self.title != title)
+            NSData *artworkData = [info objectForKey:kMRMediaRemoteNowPlayingInfoArtworkData];
```

Comment:
This is always nil on my machine. I am running macOS 10.14.3.

I have tried with iTunes, Spotify and Hermes and I never get the album art.

Reply:
That's weird, I'm also running macOS 10.14.3.

Logging info var using iTunes returns:
{
    kMRMediaRemoteNowPlayingInfoAlbum = "Whatever People Say I Am, That's What I'm Not";
    kMRMediaRemoteNowPlayingInfoAlbumiTunesStoreAdamIdentifier = 111153953;
    kMRMediaRemoteNowPlayingInfoArtist = "Arctic Monkeys";
    kMRMediaRemoteNowPlayingInfoArtistiTunesStoreAdamIdentifier = 62820413;
    kMRMediaRemoteNowPlayingInfoArtworkData = <xxxxxxxx xxxxxxxx ...>
}

On my machine: https://www.youtube.com/watch?v=BEqfOcQq_Xk